packages/core/src/components/Table/Body/Body.tsx

```diff
@@ -8,10 +9,12 @@ import { NoResultCell, NoResultRow } from './Row/Row.styled';
 import { Props } from './types';
 
 const Body: React.FC<Props> = React.memo(props => {
-    const { data, groupBy, rowIdentifier, showRowWithCardStyle, noResultRow, tableRef, withMinimap, columns } = useContext(
+    const { data, groupBy, rowIdentifier, showRowWithCardStyle, noResultRow, tableRef, withMinimap, columns, size } = useContext(
             TableComponentsCommonPropsContext
         ),
         { selectedRowIds, onRowSelection, onGroupedRowSelection, setUniqueIds, ...restProps } = props,
+        /* since minimap is positioned sticky with respect to the tbody, tbody should have full table width otherwise minimap positioning fails */
+        getTableVisibleWidth = useCallback(() => (tableRef.current ? tableRef.current.clientWidth : 0), []),
```

Comment:
You can simply use a variable over here ->>> tableClientWidth = tableRef.current?.clientWidth ?? 0;

Reply:
got it